addons/iptables.yml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
Is it possible to import this playbook snippet, "requirements.yml" or something? Then, every other Mantl playbook could depend on it.